content/docs/05.developer-guide/13.cicd/index.md

```diff
@@ -4,77 +4,220 @@ title: Create a CI/CD pipeline
+docker exec -it kestra-container-name /bin/bash
+./kestra flow validate flow_directory/myflow.yml
+./kestra flow namespace update namespace_name flow_directory/myflow.yml --no-delete
+```
 
-### Flow validation
+To validate and deploy multiple flows within a directory named `flows`, you can provide a path to that directory:
 
 ```bash
-./kestra flow validate path-to-flow-directory
+docker exec -it kestra-container-name /bin/bash
+./kestra flow validate flows/
+./kestra flow namespace update namespace_name flows/ --no-delete
 ```
 
-Validates given YAML flow files.
+The `--no-delete` flag is used to preserve existing flows already stored within your Kestra instance. Don't include that flag if you want that a specific Git repository or a local directory is used as the single source of truth for your production workflows. This way, all previously stored flows get deleted, and only those flows maintained in Git (or that directory) are kept and updated.
+
+#### CLI options
 
-**Options:**
+The CLI provides several options to customize your validation and deployment process. CLI options you should be aware of include:
 
-* `--local`: perform the validation locally using the client. Default is to use a remote call.
-* `--server`: specify the remote Kestra server URL. Default is [http://localhost:8080](http://localhost:8080).
+* `--local`: performs the validation locally using the client. By default, Kestra validates flows server-side (a remote API call to your Kestra server).
+* `--server`: allows you to specify the remote server URL. The default URL is [http://localhost:8080](http://localhost:8080).
 
-### Flow namespace update
+
+For all available CLI options on both `flow validate` and `flow namespace update` commands, use the `-h` or `--help` flag:
 
 ```bash
-./kestra flow namespace update flow-namespace-to-update path-to-flow-directory
+./kestra flow validate -h
+./kestra flow namespace update -h
 ```
 
-Deploys given YAML flow files as flows of the given namespace.
-
-**Options:**
 
-* `--no-delete`: preserve existing flows that are missing from the update. Default is to remove the flows.
-* `--server`: specify the remote Kestra server URL. Default is [http://localhost:8080](http://localhost:8080).
+#### Templates 
 
-### Template validation
+Templates can be validated and deployed in the same way as flows:
 
 ```bash
+docker exec -it kestra-container-name /bin/bash
 ./kestra template validate path-to-template-directory
+./kestra template namespace update template-namespace-to-update path-to-template-directory
+```
+
+
+### Deploy flows... from a flow!
+
+The CLI commands explained above can be used in a Bash script within a flow. This way, your CI/CD pipeline can be managed directly from your Kestra instance. 
+
+```yaml
+id: ci-cd
+namespace: prod
+variables:
+  host: "http://your_host_name:8080/" # e.g. "http://localhost:8080/"
+  auth: "yourUsername:yourPassword" # e.g. "" - leave as empty string for a locally running Kestra instance
```

Comment:
There is no auth yet on OSS so it's better to leave auth out of scope.
In EE this will be a little more complex when using OIDC.